packages/thirdweb/src/react/web/ui/ConnectWallet/screens/SendFunds.tsx

```diff
@@ -177,6 +177,7 @@ function SendFundsForm(props: {
           center="both"
           style={{
             minHeight: "200px",
+            textAlign: "center",
```

Comment:
Prefer setting the `center` prop on `Text` component that shows the error instead. We should also add `multiline` as well to fix the line-height 

```ts
<Text color="danger" center multiline >{getErrorMessage(sendTokenMutation.error)}</Text>
```